assignments/prototype-refactor.js

```diff
@@ -4,6 +4,122 @@ Prototype Refactor
 
 1. Copy and paste your code or the solution from yesterday
 
+
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+/* dimensions (These represent the character's size in the video game)
+* destroy() // prototype method that returns: `${this.name} was removed from the game.`
+*/
+
+class GameObject {
+    constructor (attributes) {
+    this.createdAt = attributes.createdAt;
+    this.name = attributes.name;
+    this.dimensions = attributes.dimensions;
```

Comment:
What is `dimensions` attribute inside of your object? and how would you access it's own attributes?